x-pack/legacy/plugins/spaces/public/views/nav_control/components/spaces_menu.tsx

```diff
@@ -175,6 +187,14 @@ class SpacesMenuUI extends Component<Props, State> {
       </EuiContextMenuItem>
     );
   };
+
+  private rendePlaceholderMenuItem = (key: string | number): JSX.Element => {
```

Comment:
Not strictly required as part of this migration, but it makes for a better user experience when loading the list of available spaces. We often take the near-immediate responses for granted when running locally, and forget that these things can take a noticeable amount of time when hosted in the real world.